src/common/InputRowRadioButtons.js

```diff
@@ -0,0 +1,154 @@
+/* @flow strict-local */
+import React, { useCallback, useRef, useMemo, useEffect } from 'react';
+import type { Node } from 'react';
+import { View } from 'react-native';
+import invariant from 'invariant';
+import { CommonActions } from '@react-navigation/native';
+import { BRAND_COLOR, createStyleSheet } from '../styles';
+
+import Touchable from './Touchable';
+import ZulipText from './ZulipText';
+import { IconRight } from './Icons';
+import type { AppNavigationProp } from '../nav/AppNavigator';
+
+type Item<TKey> = $ReadOnly<{|
+  key: TKey,
+  title: string,
+  subtitle?: string,
```

Comment:
These two should be strings waiting to be translated, right? I.e. not strings we're going to show literally to the user without passing through translation.

If so, it'd be good to make that clear in the code (and if not, to make the opposite clear). In particular, it's good for it to be clear where in the code the responsibility lies for invoking translation, so that we don't end up either failing to translate or attempting to double-translate.

I think a pretty good immediate way to do that would be to make the type `LocalizableText`. That's at least as good as any comment, plus it will prevent trying to use the string directly without translating it.

I don't think we currently have a way to ensure that a string not meant for translation -- e.g. one that's already been through translation -- accidentally gets translated. Now that I think about it, it'd probably be straightforward to do so by making `LocalizableText` an opaque type. But that's a project for another day.